include/engine/api/route_parameters.hpp

```diff
@@ -67,6 +67,16 @@ struct RouteParameters : public BaseParameters
         Full,
         False
     };
+    enum class AnnotationsType : int
+    {
+        None = 0,
+        Duration = 1 << 1,
```

Comment:
here it can start from `1<<0`, but better to use explicit  numerals 0x01, 0x02, ...